src/frontend/src/pages/Calendar/CalendarComponents/MonthSelector.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, MenuItem, TextField } from '@mui/material';
+import { Dispatch, SetStateAction } from 'react';
+import { EnumToArray, MONTH_NAMES } from '../../../utils/design-review.utils';
+
+interface MonthSelectorProps {
+  displayMonth: Date;
+  setDisplayMonth: Dispatch<SetStateAction<Date>>;
+}
+
+const MonthSelector: React.FC<MonthSelectorProps> = ({ displayMonth, setDisplayMonth }) => {
+  // TODO change this to use Pagination instead of hardocding 50 years
+  const years = [...Array(50).keys()].map((num) => (num + 2024).toString());
+
+  return (
+    <Box>
+      <TextField
+        select
+        defaultValue={displayMonth.getMonth()}
+        onChange={(event) => {
+          displayMonth.setMonth(Number(event.target.value));
+          setDisplayMonth(new Date(displayMonth));
+        }}
+      >
+        {EnumToArray(MONTH_NAMES).map((month, index) => {
+          return (
+            <MenuItem key={month} value={index}>
+              {month}
+            </MenuItem>
+          );
+        })}
+      </TextField>
+
+      <TextField
```

Comment:
The selection is covered by the menu on my screen even in full screen

Reply:
But because its selected from the menu it's not a huge deal rn if it's weird to figure out

Reply:
The menu meaning the sidebar?